app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Walrus</title>
+  <!-- <link href='https://fonts.googleapis.com/css?family=Cantarell:400italic' rel='stylesheet' type='text/css'> -->
+  <!-- <link href="CupolaRoman.ttf" rel='stylesheet' type='text/css'> -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <%= stylesheet_link_tag    'application', media: 'all' %>
+  <%= javascript_include_tag 'application' %>
+  <%= csrf_meta_tags %>
+</head>
+<body>
+  <div class= "container">
+    <div class= "header">
+      <%= image_tag ("logo.jpg"), class: "logo" %>
+      <div class= "links">
+        <a>
+          <%= link_to 'Food', sections_path%>
```

Comment:
These `link_to` tags are wrapped in `<a>` which is reduntant, a `link_to` tag generates `<a>` tags.